package.json

```diff
@@ -68,7 +68,7 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/register": "^7.12.0",
     "@babel/runtime": "^7.12.0",
-    "@cumulus/api": "7.2.1-alpha.0",
+    "@cumulus/api": "8.1.0",
     "@cumulus/aws-client": "7.0.0",
```

Comment:
We should update this as well.

Reply:
Yup, I will put up a PR to note this in the README.md as well.